src/readers/morphologyHDF5.cpp

```diff
@@ -479,6 +487,29 @@ void MorphologyHDF5::_read(const std::string& groupName,
     }
 }
 
+void MorphologyHDF5::_readEndoplasmicReticulum()
+{
+    {
+        HighFive::SilenceHDF5 silence;
+
+        try {
+            const auto group = _file->getGroup(_g_endoplasmic_reticulum);
+        } catch (const HighFive::GroupException&) {
+            return;
```

Comment:
Don't we want to emit an error message here?

Reply:
No. The endoplasmic group is optional.